components/scream/src/share/scream_pack_kokkos.hpp

```diff
@@ -254,6 +254,51 @@ smallize (const Kokkos::View<BigPack<T>*, Parms...>& vp) {
   return repack<SCREAM_SMALL_PACK_SIZE>(vp);
 }
 
+//
+// Take an array of Host scalar pointers and turn them into device pack views
+//
+template <Int N, typename Scalar, typename PackT, typename DeviceT>
```

Comment:
Could default `DeviceT=DefaultDevice`.

Reply:
Good point. What I'd really like is for the compiler to infer the Device type from the View argument.